pkg/build/api/types.go

```diff
@@ -142,6 +146,9 @@ const (
 
 	// BuildPhaseCancelled indicates that a running/pending build was stopped from executing.
 	BuildPhaseCancelled BuildPhase = "Cancelled"
+
+	// BuildPhaseSkipped indicates that a queued build was skipped.
+	BuildPhaseSkipped BuildPhase = "Skipped"
```

Comment:
Regarding 7856, yes we are considering it, but we won't remove phase from
any of our resources or now.  Be aware that you cannot add a new phase in a
backwards compatible way (it would break old clients that expect phases),
you can only add new conditions.  So this new change couldn't add a new
phase.

On Mon, Apr 11, 2016 at 8:06 AM, Michail Kargakis <notifications@github.com>
wrote:

> In pkg/build/api/types.go
> <https://github.com/openshift/origin/pull/8453#discussion_r59193077>:
>
> > @@ -142,6 +146,9 @@ const (
> >
> >  	// BuildPhaseCancelled indicates that a running/pending build was stopped from executing.
> >  	BuildPhaseCancelled BuildPhase = "Cancelled"
> > +
> > +	// BuildPhaseSkipped indicates that a queued build was skipped.
> > +	BuildPhaseSkipped BuildPhase = "Skipped"
>
> @smarterclayton <https://github.com/smarterclayton> are we considering
> kubernetes/kubernetes#7856
> <https://github.com/kubernetes/kubernetes/issues/7856> for OS resources?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/8453/files/0377afaa8ee7bb504a6e7484892b1dbeb5be3e41#r59193077>
>